packages/lesswrong/components/quickTakes/QuickTakesEntry.tsx

```diff
@@ -120,118 +87,35 @@ const QuickTakesEntry = ({
   defaultFocus?: boolean,
   submitButtonAtBottom?: boolean,
   className?: string,
-  editorClassName?: string,
-  tagsClassName?: string,
-  buttonClassName?: string,
   successCallback?: CommentSuccessCallback,
   cancelCallback?: CommentCancelCallback,
   classes: ClassesType<typeof styles>,
 }) => {
-  const editorType = "ckEditorMarkup";
-  const editorRef = useRef<EditorType>(null);
-  const formRef = useRef<HTMLFormElement>(null);
+  const ref = useRef<HTMLDivElement>(null);
   const [expanded, setExpanded] = useState(defaultExpanded);
-  const [loadingSubmit, setLoadingSubmit] = useState(false);
-  const [contents, setContents] = useState(() => getInitialEditorContents(
-    undefined,
-    null,
-    "contents",
-    currentUser,
-  ));
   const {
-    loading: loadingTags,
     frontpage,
     selectedTagIds,
-    tags,
-    frontpageTagId,
-    onTagSelected,
-    onTagRemoved,
   } = useQuickTakesTags();
-  const {create} = useCreate({
-    collectionName: "Comments",
-    fragmentName: "ShortformComments",
-  });
-
-  const onChange = useCallback(({contents}: EditorChangeEvent) => {
-    setContents(contents);
-  }, []);
-
-  const lastSubmittedAt = useRef(0);
-
-  const onSubmit = useCallback(async (ev?: MouseEvent) => {
-    ev?.preventDefault();
-
-    // Prevent accidental double submits
-    if (Date.now() - lastSubmittedAt.current < 1000) {
-      return;
-    }
-    lastSubmittedAt.current = Date.now();
-
-    setLoadingSubmit(true);
-    try {
-      const contents = await editorRef.current?.submitData();
-      const response = await create({
-        data: {
-          shortform: true,
-          shortformFrontpage: frontpage,
-          relevantTagIds: selectedTagIds,
-          // There's some magic that makes this work even though it is missing
-          // some fields that are marked as required. It hard to work out exactly
-          // what's going on without getting lost in the seas of `any`.
-          // @ts-ignore
-          contents,
-        },
-      });
-      const comment = response.data?.createComment.data;
-      void successCallback?.(comment, {form: formRef.current});
-      editorRef.current?.clear(currentUser);
-    } catch (e) {
-      // eslint-disable-next-line no-console
-      console.error(e);
-      void cancelCallback?.(e);
-    }
-    setLoadingSubmit(false);
-  }, [
-    create,
-    frontpage,
-    selectedTagIds,
-    successCallback,
-    cancelCallback,
-    currentUser,
-  ]);
 
   const onCancel = useCallback(async (ev?: MouseEvent) => {
     ev?.preventDefault();
     setExpanded(false);
     void cancelCallback?.();
   }, [cancelCallback]);
```

Comment:
I noticed that clicking "Cancel" doesn't clear the input, just collapses it. I think it's not blocking for this PR but we should probably add that to asana or something since it's a bit surprising.